schutzbot/scheduled_cloud_cleaner.sh

```diff
@@ -50,7 +49,41 @@ echo "There are resources from $TO_DELETE_COUNT test runs to delete."
 
 for i in $(seq 0 $(("$TO_DELETE_COUNT"-1))); do
     echo "Running cloud-cleaner in Azure for resources with TEST_ID: ${RESOURCE_TO_DELETE_LIST[$i]}"
-    TEST_ID=${RESOURCE_TO_DELETE_LIST[$i]} /usr/libexec/osbuild-composer-test/cloud-cleaner
+    # TODO uncomment
+    # TEST_ID=${RESOURCE_TO_DELETE_LIST[$i]} /usr/libexec/osbuild-composer-test/cloud-cleaner
+done
+
+# Explicitly check the other storage accounts
+STORAGE_ACCOUNT_LIST=$(az resource list -g "$AZURE_RESOURCE_GROUP" --resource-type Microsoft.Storage/storageAccounts)
+STORAGE_ACCOUNT_COUNT=$(echo "$STORAGE_ACCOUNT_LIST" | jq .[].name | wc -l)
+DELETE_TIME=$(date -d "- $HOURS_BACK hours" +%s)
+for i in $(seq 0 $(("$STORAGE_ACCOUNT_COUNT"-1))); do
+    STORAGE_ACCOUNT_NAME=$(echo "$STORAGE_ACCOUNT_LIST" | jq .["$i"].name | tr -d '"')
+    OLD_BLOBS_LIST_NAMES=()
+    echo "Checking storage account $STORAGE_ACCOUNT_NAME for old blobs."
+    CONTAINER_LIST=$(az storage container list --account-name "$STORAGE_ACCOUNT_NAME")
+    CONTAINER_COUNT=$(echo "$CONTAINER_LIST" | jq .[].name | wc -l)
+    for i2 in $(seq 0 $(("$CONTAINER_COUNT"-1))); do
+        CONTAINER_NAME=$(echo "$CONTAINER_LIST" | jq .["$i2"].name | tr -d '"')
+        BLOB_LIST=$(az storage blob list --account-name "$STORAGE_ACCOUNT_NAME" --container-name "$CONTAINER_NAME")
+        BLOB_COUNT=$(echo "$BLOB_LIST" | jq .[].name | wc -l)
+        for i3 in $(seq 0 $(("$BLOB_COUNT"-1))); do
+            OLD_BLOBS_LIST_NAMES+=("$(echo "$BLOB_LIST" | jq .["$i3"].name | sed -e 's/^[^-]*-//' | tr -d '"')")
+        done
+    done
+
+    mapfile -t BLOBS_TO_DELETE_LIST  < <(printf "%s\n" "${OLD_BLOBS_LIST_NAMES[@]}" | sort -u)
+    TO_DELETE_COUNT=${#BLOBS_TO_DELETE_LIST[@]}
+    if [ ${#OLD_BLOBS_LIST_NAMES[@]} == 0 ]; then
+        echo "No blobs delete in $STORAGE_ACCOUNT_NAME."
+    fi
+    echo "Deleting $TO_DELETE_COUNT blobs in $STORAGE_ACCOUNT_NAME."
+
+    for i2 in $(seq 0 $(("$TO_DELETE_COUNT"-1))); do
+        echo "Running cloud-cleaner in Azure for resources with TEST_ID: ${RESOURCE_TO_DELETE_LIST[$i2]}"
+        # TODO uncomment in actual test run
+        # AZURE_STORAGE_ACCOUNT="$STORAGE_ACCOUNT_NAME" TEST_ID=${RESOURCE_TO_DELETE_LIST[$i]} /usr/libexec/osbuild-composer-test/cloud-cleaner
```

Comment:
@jrusz right but I thought setting the AZURE_STORAGE_ACCOUNT here will overwrite whatever we get from the CI environment; and set it from travisuploadci to one of the scanned ones.

Reply:
Oh I see it now, well that should work then...

Reply:
If you think it's an ok approach overall I'll enable this here as a test run.